lib/python/pyflyby/_util.py

```diff
@@ -2,14 +2,13 @@
 # Copyright (C) 2011, 2012, 2013, 2014, 2015, 2018 Karl Chen.
 # License: MIT http://opensource.org/licenses/MIT
 
-from __future__ import (absolute_import, division, print_function,
-                        with_statement)
+
 
 from   contextlib               import contextmanager
 import inspect
 import os
 import six
-from   six                      import PY3, reraise
+from   six                      import reraise
 import sys
 import types
 
```

Comment:
Not sure if the below is still required

Reply:
You mean: 

```python
# Python 2/3 compatibility
DictProxyType = type(object.__dict__)
```

I don't know either, haven't found that branch (where it's being used) being called in any of the tests. I would still keep it for now, as I am not sure about it.